justspacesapp/urls.py

```diff
@@ -13,11 +13,6 @@
 import fobi.views
 
 urlpatterns = [
-    # placeholder view for a public-facing landing page
```

Comment:
@beamalsky - are we okay removing this placeholder now? I know it served a purpose for the demo site for UCD, i.e., getting content on a home page. But it seems like we can do away with it?

Reply:
I think that's a good approach @jeancochrane. Could you do that on this PR? And when we're ready, we can remove the placeholder, once-and-for-all.